src/demo/js/script.js

```diff
@@ -179,8 +179,55 @@ window.addEventListener(
   },
   false
 );
+function exportPhp() {
+  let properties = [
+    "border",
+    "stroke",
+    "ring",
+    "fire",
+    "currStreakNum",
+    "sideNums",
+    "currStreakLabel",
+    "sideLabels",
+    "dates",
+    "background",
+  ];
+  let array = Array.from(document.querySelectorAll(".param")).reduce(
+    (acc, next) => {
+      let obj = { ...acc };
+      let value = next.value;
+      if (value.indexOf("#") >= 0) {
+        // if the value is colour, remove the hash sign
+        value = value.replace(/#/g, "");
+        if (value.length > 6) {
+          // if the value is in hexa and opacity is 1, remove FF
+          value = value.replace(/(F|f){2}$/, "");
+        }
+      }
+      obj[next.id] = value;
+      return obj;
+    },
+    {}
+  );
+  var exportPhp = {};
+
+  for (const [key, val] of Object.entries(array)) {
+    if (properties.includes(key) > 0) {
+      exportPhp[key] = val;
+    }
+  }
+
+  const output =
+    "[\n" +
+    Object.keys(exportPhp)
+      .map((key) => `\t"${key}" => "#${exportPhp[key]}",\n`)
+      .join("") +
+    "]";
```

Comment:
`exportPhp` is a function name so should not also be a variable. You should use a different name such as `params`.

```suggestion
  const output =
    "[\n" +
    Object.keys(params)
      .map((key) => `\t"${key}" => "#${params[key]}",\n`)
      .join("") +
    "]";
```